moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -226,6 +226,15 @@ class Robot
                                   const std::vector<double> & to,
                                   double t) const;
 
+
+  /**
+  * @brief getCurrentRobotState - Returns a vector<double> of the
+  * current joint positions of the robot from robot_state_.
+  *
+  * @return std::vector<double> current_joint_position
+  */
+  std::vector<double> getCurrentRobotState(void) const;
```

Comment:
Let's call this `getJointState`.  I know MoveIt calls it `RobotState`, but that's confusing to a more traditional robot programmer.